src/app/chip_data_model.cmake

```diff
@@ -66,20 +66,35 @@ endfunction()
 #
 # Configure ${APP_TARGET} based on the selected data model configuration.
 # Available options are:
-#   SCOPE           CMake scope keyword that defines the scope of included sources.
-#                   The default is PRIVATE scope.
-#   BYPASS_IDL      Bypass code generation from .matter IDL file.
-#   ZAP_FILE        Path to the ZAP file, used to determine the list of clusters
-#                   supported by the application.
-#   IDL             .matter IDL file to use for codegen. Inferred from ZAP_FILE
-#                   if not provided
-#   EXTERNAL_CLUSTERS Clusters with external implementations. The default implementations
-#                   will not be used nor required for these clusters.
-#                   Format: MY_CUSTOM_CLUSTER'.
+# SCOPE             CMake scope keyword that defines the scope of included sources.
+# The default is PRIVATE scope.
```

Comment:
Formatting looks a bit odd

Reply:
It is due to alignment with the upstream.